templates/operations.ts.hbs

```diff
@@ -186,8 +187,11 @@
           method: "{{loud method}}"
         }
       );
+      if(returnTypeParam && returnTypeParam === returnType.REQUEST) {
+        return req
+      }
       const response = await fetch(req);
-      if (rawResponse) {
+      if (rawResponse || (returnTypeParam && returnTypeParam === returnType.RESPONSE)) {
```

Comment:
```suggestion
      if (returnTypeParam === returnType.RESPONSE || (rawResponse && returnTypeParam !== returnType.JSON)) {
```
If somebody does `someMethod({ returnTypeParam: returnType.JSON }, true)`, then `returnTypeParam` should take precedence over `rawResponse`.